### DISCUSSION TITLE
Gender classification in dicebar

### DISCUSSION THREAD

**Discussion by grtsnx** [dicebear/dicebear#425] [Ideas]
It would be nice if there could be gender classification which could be passed as a parameter in the HTTP-API .

This way not all generated avatars would be random but can be random and gender based.

**Comment by FlorianKoerner** [comment]
Hi @thegrtnx, thanks for your suggestion.

The discussion is not entirely new. In principle, avatars have no gender. Only the viewer himself associates a gender with the avatar shown. However, this association is very much dependent on cultural background.

The association is always based on small details, such as hair. In some regions, men tend to have short hair, while in others they tend to have longer hair. The same goes for women. Women don't always have long hair either. Reddish colours are not necessarily feminine and bluish colours are not necessarily masculine. It's always a matter of culture. And there are also women with beards.

It's a difficult subject. That's why I decided to stop offering gender filters. But that shouldn't stop you from creating avatars that you think look male or female. There are already plenty of options available to help you create avatars that look more masculine or feminine to your users.

Feel free to use the [Playground](https://www.dicebear.com/playground/) to create custom API endpoints. For example, if you want to use the [adventurer](https://www.dicebear.com/styles/adventurer/) avatar style and associate women with long hair and men with short hair, then these are the two API endpoints for you:

**Women**
https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26

![](https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26&size=128&seed=1) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26&size=128&seed=2) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26&size=128&seed=3) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26&size=128&seed=4) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=long01,long02,long03,long04,long05,long06,long07,long08,long09,long10,long11,long12,long13,long14,long15,long16,long17,long18,long19,long20,long21,long22,long23,long24,long25,long26&size=128&seed=5)

**Men**
https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19

![](https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19&size=128&seed=1) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19&size=128&seed=2) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19&size=128&seed=3) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19&size=128&seed=4) ![](https://api.dicebear.com/9.x/adventurer/svg?hair=short01,short02,short03,short04,short05,short06,short07,short08,short09,short10,short11,short12,short13,short14,short15,short16,short17,short18,short19&size=128&seed=5)

**See also**
https://github.com/search?q=repo%3Adicebear%2Fdicebear+gender&type=issues

**Comment by grtsnx** [comment]
Alright.

Thanks for your help. @FlorianKoerner
